File: starkaidautomacao/app/src/main/java/com/starkaid/starkaidapp/models/ComandoSocial.kt
package com.starkaid.starkaidapp.models

import com.google.gson.annotations.SerializedName

data class ComandoSocial(
    @SerializedName("id") val id: String,
    @SerializedName("userId") val userId: String,
    @SerializedName("comando") val comando: String,
    @SerializedName("resposta") val resposta: String,
    @SerializedName("respostasAleatorias") val respostasAleatorias: String? // JSON string
)

File: StarkAid.WindowsForms/Models/ComandoSocial.cs
namespace StarkAid.WindowsForms.Models;

public class ComandoSocial
{
    public Guid Id { get; set; }
    public string Comando { get; set; } = string.Empty;
    public string Resposta { get; set; } = string.Empty;
    public string? RespostasAleatorias { get; set; }
}

